packages/manager/modules/anthos/src/dashboard/host/host-table/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,17 @@
+{
+  "anthos_dashboard_host_id": "ID",
+  "anthos_dashboard_host_name": "Nom",
+  "anthos_dashboard_host_profile": "Profil",
+  "anthos_dashboard_host_rack": "Rack",
+  "anthos_dashboard_host_status": "Statut",
+  "anthos_dashboard_host_stateful": "Stateful",
+  "anthos_dashboard_host_status_BAREMETAL_STATUS_UNSPECIFIED": "Inconnu",
+  "anthos_dashboard_host_status_BAREMETAL_STATUS_DELIVERING": "Livraison en cours",
+  "anthos_dashboard_host_status_BAREMETAL_STATUS_AVAILABLE": "Disponible",
+  "anthos_dashboard_host_status_BAREMETAL_STATUS_IN_USE": "Utilisé",
+  "anthos_dashboard_host_status_BAREMETAL_STATUS_IN_MAINTENANCE": "En cours de maintenance",
```

Comment:
We could've just concat `anthos_dashboard_host_status_BAREMETAL_STATUS_${host.status}`

Reply:
Actually, `host.status` is `BAREMETAL_STATUS_IN_MAINTENANCE` or `BAREMETAL_STATUS_IN_USE`... It comes from the API